deploy/addons/gvisor/config.toml

```diff
@@ -0,0 +1,69 @@
+root = "/var/lib/containerd"
```

Comment:
It seems strange that the default config.toml is included in the gvisor addon and that we restore this file when the addon is disabled. Shouldn't the addon enable process backup the existing config.toml and restore that?

Reply:
that makes sense, done!